plot_data/core.py

```diff
@@ -89,10 +92,44 @@ def mpl_plot(self, ax=None):
         return ax
 
 
+class Figure(PlotDataObject):
+    """ Abstract interface for handling html exportable objects in module. """
+
+    def __init__(self, type_: str, width: int = 750, height: int = 400, name: str = '', **kwargs):
+        self.width = width
+        self.height = height
+        PlotDataObject.__init__(self, type_=type_, name=name, **kwargs)
+
+    @property
+    def template(self):
+        return getattr(templates, self._template_name)
+
+    def _export_formats(self) -> List[ExportFormat]:
```

Comment:
Do we need an export format for this ? Export format mostly is a platform feature

Reply:
I don't know. I just copied what was done for json and others. Maybe we should have a short talk about it.

Reply:
Is it ok like that ? (following our talk)